projects/demo/src/modules/app/pages.ts

```diff
@@ -925,6 +925,12 @@ export const pages: TuiDocPages = [
         keywords: `чек, радио, ввод, форма, form, checkbox, radio, toggle`,
         route: `/experimental/toggle`,
     },
+    {
+        section: `Experimental`,
+        title: `Card `,
```

Comment:
We have renamed old Card to ThumbnailCard so no space is necessary ;)

Reply:
fixed